konrad/cloud.py

```diff
@@ -338,15 +338,21 @@ class DirectInputCloud(Cloud):
     }
 
     def __init__(self, numlevels, cloud_fraction, lw_optical_thickness,
-                 sw_optical_thickness, forward_scattering_fraction=0,
-                 asymmetry_parameter=0.85, single_scattering_albedo=0.9,
-                 norm_index=None):
+                 sw_optical_thickness, coupling='convection',
+                 forward_scattering_fraction=0, asymmetry_parameter=0.85,
+                 single_scattering_albedo=0.9, norm_index=None):
 
         """Define a cloud based on properties that are directly used by the
         radiation scheme, namely cloud optical depth and scattering parameters.
 
         Parameters:
             numlevels (int): Number of atmospheric levels.
+            coupling (str): Mechanism with which the cloud is coupled to the
+                atmospheric profile:
+                    * 'convection': convective top
+                    * 'temperature': freezing level
+                    * 'subsidence': level of maximum subsidence divergence
+                    * 'pressue': fixed at pressure (no coupling)
```

Comment:
I'd think I'd make the choices a bit more obvious, eg using ``'freezing_level'`` and ``'convective_top'`` instead of ``'temperature'`` and ``'convection'``. Also you have a typo in 'pressure'